src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -71,40 +44,74 @@
 		</div>
 
 		<!-- edit entry -->
-		<div v-if="item.edit" class="app-navigation-entry-edit">
+		<div v-if="editable" class="app-navigation-entry-edit">
 			<form @submit.prevent.stop="item.edit.action">
-				<input :placeholder="item.edit.text" type="text">
+				<input placeholder="placeholder" type="text">
 				<input type="submit" value="" class="icon-confirm">
 				<input type="submit" value="" class="icon-close"
 					@click.stop.prevent="cancelEdit">
 			</form>
 		</div>
 
-		<!-- if the item has children, inject the component with proper data -->
-		<ul v-if="item.children">
-			<app-navigation-item v-for="(child, key) in item.children" :key="key" :item="child" />
+		<!-- Icon or image from url -->
+		<a :class="[ !isIconUrl && bullet === '' ? icon : '' ]" href="#"
+			@click="onClick">
+			<img v-if="isIconUrl && bullet === ''" :src="icon">
+			{{ title }}
+		</a>
+
+		<!-- Popover, counter and button(s) -->
+		<div v-if="this.$slots.counter || this.$slots.actions" class="app-navigation-entry__utils">
+			<ul>
+				<slot name="counter" />
+				<slot name="actions" />
+			</ul>
+		</div>
+
+		<!-- Children elements -->
+		<ul v-if="canHaveChildren && this.$slots.children ">
+			<slot name="children" />
 		</ul>
 	</nav-element>
 </template>
 
 <script>
-import { PopoverMenu } from 'Components/PopoverMenu'
 import ClickOutside from 'vue-click-outside'
-import { AppNavigationCounter } from 'Components/AppNavigationCounter'
-
 export default {
 	name: 'AppNavigationItem',
 	components: {
-		PopoverMenu,
-		AppNavigationCounter
 	},
 	directives: {
 		ClickOutside
 	},
 	props: {
+		/**
+		 * The title of the element.
+		 */
+		title: {
+			type: String,
+			required: true
+		},
+		/**
+		* Passing in a route will make the root element of this
+		* component a <router-link /> that points to that route.
+		* By leaving this blank, the root element will be a <li>.
+       	*/
+		to: {
+			type: String,
+			default: ''
+		},
+		/**
+		* Pass in `false` if you want the matching behaviour to
+		* be inclusive: https://router.vuejs.org/api/#exact
+       	*/
+		exactRoute: {
+			type: Boolean,
+			default: true
```

Comment:
```suggestion
			default: false
```
https://router.vuejs.org/api/#exact